frontend/src/utils/DropDownAutoCompleteTextArea.svelte

```diff
@@ -0,0 +1,89 @@
+<style>
+  .dropdown {
+    text-align: left;
+    position: absolute;
+    margin-top: 3px;
+  }
+
+  textarea {
+    background: inherit;
+    color: inherit;
+    border: none;
+    border-bottom: 1px solid white;
+    flex-grow: 1;
+    height: 10em;
+    margin-left: 1ch;
+  }
+
+  textarea:focus {
+    outline: none;
+    box-shadow: inset 0 -1px 0 var(--main-fg-color);
+  }
+</style>
+
+<script>
+  import { onMount } from "svelte";
+  import DropDown from "./DropDown.svelte";
+
+  export let comment, options, pattern;
+  let autoOptions = options;
+  let dropdown, string, input;
+
+  document.onkeyup = handle;
```

Comment:
I think this will cause problems if keys are pressed outside of this context. We probably only want to set event handlers on elements in the scope of this component.